main.go

```diff
@@ -254,6 +257,19 @@ func main() {
 		envString("", "GITSYNC_ASKPASS_URL", "GIT_SYNC_ASKPASS_URL", "GIT_ASKPASS_URL"),
 		"a URL to query for git credentials (username=<value> and password=<value>)")
 
+	flGithubBaseURL := pflag.String("github-base-url",
+		envString("https://api.github.com/", "GITSYNC_GITHUB_BASE_URL"),
+		"the GitHub base URL to use when making requests to GitHub when using app auth")
+	flGithubAppPrivateKeyFile := pflag.String("github-app-private-key-file",
```

Comment:
See #886 for how this can be offered both as an env ($GITSYNC_GITHUB_APP_PRIVATE_KEY) and as a file as you have here

Reply:
Thanks, I'll take a look at that now :)